docs/pages/eas/environment-variables.mdx

```diff
@@ -8,7 +8,7 @@ import { Terminal } from '~/ui/components/Snippet';
 
 [Environment variables in Expo](/guides/environment-variables) describe how to use environment variables with the Expo framework and **.env** files to set environment variables that can be inlined in your JavaScript code. Expo CLI will substitute prefixed variables in your code (for example, `process.env.EXPO_PUBLIC_VARNAME`) with the corresponding environment variable values in **.env** files on your development machine.
 
-Since EAS Build and Workflows jobs run on a remote server, **.env** files may not be available. For instance, if **.env** files are excluded from your project because they are listed in **.gitignore** or not committed to your local version control system.
```

Comment:
i think it's ok to say EAS Build and Workflows here, since the EAS carries over from the first to the second. EAS Build and Submit is something we'd commonly say for example